website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Community
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'n/a'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/postgres-configs'
+---
+
+:::info Community plugin
+
+Some core functionality may be limited.
+
+:::
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta}/>
+
+
+## Profile Configuration
+
+CrateDB targets should be set up using the following configuration in your `profiles.yml` file.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+company-name:
+  target: dev
+  outputs:
+    dev:
+      type: cratedb
+      host: [hostname]
+      user: [username]
+      password: [password]
+      port: [port]
+      dbname: [database name] # or database instead of dbname
+      schema: [dbt schema]
+      threads: [optional, 1 or more]
+      [keepalives_idle](#keepalives_idle): 0 # default 0, indicating the system default. See below
+      connect_timeout: 10 # default 10 seconds
+      [retries](#retries): 1  # default 1 retry on error/timeout when opening connections
+      [search_path](#search_path): [optional, override the default postgres search_path]
+      [role](#role): [optional, set the role dbt assumes when executing queries]
+      [sslmode](#sslmode): [optional, set the sslmode used to connect to the database]
+      [sslcert](#sslcert): [optional, set the sslcert to control the certifcate file location]
+      [sslkey](#sslkey): [optional, set the sslkey to control the location of the private key]
+      [sslrootcert](#sslrootcert): [optional, set the sslrootcert config value to a new file path in order to customize the file location that contain root certificates]
+  
+```
+
+</File>
+
+### Configuration notes
+
+This adapter strongly depends on dbt-postgres documented at [Postgres profile setup](/docs/core/connect-data-platform/postgres-setup). CrateDB targets are configured exactly the same way.
+
+
+### Configuration differences
+
+#### search_path
+
+The `search_path` config controls the CrateDB "search path" that dbt configures
+when opening new connections to the database. By default, the CrateDB search
+path is `"doc"`, meaning that unqualified <Term id="table" /> names will be
+searched for in the `doc` schema.
```

Comment:
That little section at the bottom outlines the **typical differences** of CrateDB. Please also share your opinion about what you would put in here.

Reply:
Maybe any of those would also be relevant to replicate here, in order to educate users approaching CrateDB the very first time from the dbt perspective?

- CrateDB’s fixed catalog name is `crate`, the default schema name is `doc`.
- CrateDB does not implement the notion of a database, however tables can be created in different [schemas](https://cratedb.com/docs/crate/reference/en/latest/general/ddl/create-table.html#ddl-create-table-schemas).
- When asked for a database name, specifying a schema name (any), or the fixed catalog name `crate` may be applicable.
- If a database-/schema-name is omitted while connecting, the PostgreSQL drivers may default to the “username”.
- The predefined [superuser](https://cratedb.com/docs/crate/reference/en/latest/admin/user-management.html#administration-user-management) on an unconfigured CrateDB cluster is called `crate`, defined without a password.
- For authenticating properly, please learn about the available [authentication](https://cratedb.com/docs/crate/reference/en/latest/admin/auth/index.html#admin-auth) options.

-- https://cratedb.com/docs/crate/clients-tools/en/latest/connect/#configure

Reply:
Let's not duplicate the information and simply have the link to our documentation (easier to update if we decide to change something)

Reply:
I absolutely agree. Let's do it like [Greenplum setup](https://docs.getdbt.com/docs/core/connect-data-platform/greenplum-setup) is doing it:

> For further and more likely up-to-date information, see the documentation about [using dbt with CrateDB](https://cratedb.com/docs/guide/integrate/dbt/).

Reply:
I've removed all too specific information, and will be adding it to [using dbt with CrateDB](https://cratedb.com/docs/guide/integrate/dbt/) instead. I've also updated the document body to include a relevant link to that page.